.github/workflows/performance-benchmarks.yml

```diff
@@ -47,9 +47,25 @@ jobs:
       - name: Download the thing
         working-directory: e2e/cypress
         run: |
-          curl https://pr-builds.mattermost.com/mattermost-webapp/commit/${{ github.event.pull_request.head.sha || github.sha }}/mattermost-enterprise-linux-amd64.tar.gz > mattermost.tar.gz
-          tar -xvf mattermost.tar.gz
+          if [[ "${{ github.event.pull_request.head.sha }}" != "" ]]; then
+            echo "Using server build from pr-builds.mattermost.com"
+            curl https://pr-builds.mattermost.com/mattermost-webapp/commit/${{ github.event.pull_request.head.sha }}/mattermost-enterprise-linux-amd64.tar.gz > mattermost.tar.gz
 
+            tar xf mattermost.tar.gz
```

Comment:
Probably nothing, but I noticed you used to call `tar -xvf` and now you are calling `tar xf`.

Reply:
Yeah, that was intentional. I wanted to simplify the output and match what the server calls here. I was kind of surprised to find that the `-` was unnecessary though